### DISCUSSION TITLE
Usage of the Hy logo?

### DISCUSSION THREAD

**Discussion by matkuki** [hylang/hy#2516] [Q&A] [answered]
Hi,

I would like to integrate `Hy` into my editor: [ExCo](https://github.com/matkuki/ExCo).
Can I use the `Cuddles` logo or the other yellow cuttlefish logo (`hy-logo-small.png`) to show something like:
`Powered by Hy` or `Hy programming language supported`?

Thanks
Matic

**Comment by Kodiologist** [comment]
Yeah, the images in the Hy repository should be covered by its license, and the license is Expat, which is [compatible with the GPL](https://www.gnu.org/licenses/license-list.en.html#Expat).

**Reply by Kodiologist** [reply, marked as answer]
Correction: the logo and Cuddles are covered under two Creative Commons licenses; see https://github.com/hylang/hy/blob/d4357fe35c2540fc5cb03796085b26f5fcbe7e91/docs/_static/hy_logo-about.txt
